docs/paper.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "antimeridian: A Python package for correcting geometries that cross the 180th meridian"
+tags:
+  - Python
+  - geospatial
+  - antimeridian
+authors:
+  - name: Peter Gadomski
+    orcid: 0000-0003-4877-7217
+    corresponding: true
+    affiliation: 1
+  - name: Preston Hartzell
+    orcid: 0000-0002-8293-3706
+    affiliation: 2
+affiliations:
+  - name: Development Seed, USA
+    index: 1
+  - name: Element 84, Inc., USA
+    index: 2
+date: 10 October 2024
+bibliography: paper.bib
+---
+
+## Summary
+
+Locations on and around planet Earth are commonly represented in a geodetic coordinate system with a longitude, a latitude, and a height.
+Longitude is the "horizontal" dimension with a domain from -180° to 180°, which spans the entire 360° circumference of the planet. Where the two domain bounds meet is known as the _180th meridian_ or the _antimeridian_.
+
+![Earth map centered on the Pacific ocean, with the 180th meridian highlighted.](./img/antimeridian.jpg)
+
+The GeoJSON specification [@10.17487/RFC7946] describes how antimeridian-crossing shapes should be split into multiple shapes at the 180th meridian.
+Real-world geometries often do not comply with the specification, often due to projected coordinates being naively reprojected to geodetic coordinates systems.
+This leads confusing and unrepresentable geometries.
+Our **antimeridan** package provides Python functions for correcting improper geometries, as well as other related utilities.
+
+## Statement of need
+
+Because of the relative lack of populated settlements on the other side of the world, the proliferation of British maps in the late 19th century, and more, the Prime Meridian (0°) runs through Greenwich, England [@alma992356353405961].
```

Comment:
```suggestion
Because of factors such as the relative lack of populated settlements along the 180th meridian and the proliferation of British maps in the late 19th century, the Prime Meridian (0° longitude) runs through Greenwich, England [@alma992356353405961].
```